plugins/catalog/src/components/CatalogTable/CatalogTable.tsx

```diff
@@ -163,6 +164,8 @@ export const CatalogTable = (props: CatalogTableProps) => {
     typeColumn.hidden = !showTypeColumn;
   }
   const showPagination = rows.length > 20;
+  const optionsOverride =
```

Comment:
Don't think you need to do this checking here, typescript is going to take care of making sure that this has something, and I think it's fine to spread null into an object anyways.

You should just be able to spread `options` in below.

Reply:
Updated :)